src/global_logic.js

```diff
@@ -730,6 +733,7 @@ module.exports.calcBasedOneSummon = function (summonind, prof, buff, totals) {
         coeffs["LBKonshin"] = LBKonshinCoeff;
         coeffs["other"] = otherCoeff;
         coeffs["hpRatio"] = hpCoeff;
+        coeffs["ougiGageBuff"] = ougiGageBuff - 1.0;
```

Comment:
Removing -1.0 can be avoided by moving adding buff["ougiGage"] parameter to line 613 without any side issues.

Reply:
Like this:
var ougiGageBuff = + totals[key]["ougiGageBuff"] + (0.01 * totals[key]["LB"]["OugiGageBuff"]) - totals[key]["ougiDebuff"];
var expectedOugiGage = ougiGageBuff + buff["ougiGage"] ;
.
.
.
coeffs["ougiGageBuff"] = ougiGageBuff; // No need - 1.0 anymore

Reply:
ougiGageBuff is used in line 616 & 618.
Problems occur if I change it.
https://github.com/kagekiyo7/motocal/blob/Add-CharaLB/src/global_logic.js#L616

Reply:
Oops I miss that. The reason you choose -1 was it is default value of buff["ougiGage"]? If so is there any case where that value isn't 1?

Reply:
>The reason you choose -1 was it is default value of buff["ougiGage"]? 

yes. https://github.com/kagekiyo7/motocal/blob/Add-CharaLB/src/global_logic.js#L1196

Reply:
>If so is there any case where that value isn't 1?

The value 1 is to align the number of digits.
it must be coeff. (For example: 0.1 (+10%))